features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.types.listing/listing.hbs

```diff
@@ -39,6 +39,10 @@
     <script type="text/javascript">
         {{! WARN: DO NOT BLINDLY PASS WHOLE CONTEXT HERE}}
         var clientJsAppContext = "{{{@app.context}}}";
+        var deviceTypesList = [ {{#each deviceTypesList}}
+            { "deviceTypeLabel" : "{{deviceTypeLabel}}", "deviceTypeId" : "{{deviceTypeId}}", "deviceTypeName" : "{{deviceTypeName}}", "hasCustTemplate" : {{hasCustTemplate}} },
+        {{/each}}
+        ];
```

Comment:
Yes, It is because "hasCustTemplate" is a boolean. Adding quotes will convert it into a String forcefully.